### ISSUE TITLE
Exclude directories from workspace for crawling performance reasons

### ISSUE THREAD

**Issue by Thiruppathi** [mathieudutour/foam-gatsby-template#10]
```json 
 "files.exclude": {
    "**/node_modules": true
  },
  "files.watcherExclude": {
    "**/node_modules": true
  },
```

Reference From: https://github.com/foambubble/foam/issues/130

**Comment by Am3ra** [comment]
isn't this resolved by  #11 ?

**Comment by mathieudutour** [comment]
Yep 👍
